userguide/advanceduse/ssh.rst

```diff
@@ -43,20 +43,25 @@ If you are using a Linux-based phone (like the Pinephone)::
 
     sudo service ssh start
 
+To have the ssh server start after every boot on a Linux-based phone::
```

Comment:
"Linux-based phone" seems unclear to me. Android is Linux-based, and so is Ubuntu Touch... everywhere.